client/X11/xf_input.c

```diff
@@ -875,17 +877,11 @@ static int xf_input_handle_event_remote(xfContext* xfc, const XEvent* event)
 				/* fallthrough */
 				WINPR_FALLTHROUGH
 			case XI_TouchUpdate:
-				/* fallthrough */
-				WINPR_FALLTHROUGH
```

Comment:
what not working here?

Reply:
Well it seems like for gcc a fallthrough is when you have
```C
case item:
    ... code...
   <no break>
case item 2:
```
The fallthrough is only at the point where it's missing a break, other cases are just regular `switch/case` matches (ie when there's no code between `case`)

Reply:
Looks like this is actually the standard way of using fallthrough: https://en.cppreference.com/w/c/language/attributes/fallthrough